images/sriovdp-daemonset.yaml

```diff
@@ -21,6 +21,7 @@ spec:
         tier: node
         app: sriovdp
     spec:
+      hostPID: true
```

Comment:
I didn't see why this is solving the problem of bind address already in use, could you please explain it a little bit more?

Reply:
Please see https://github.com/intel/sriov-network-device-plugin/pull/34#discussion_r232981011

Reply:
I now have a suspicion that this has something to do with the way we deploy our clusters. I experience permission issues with SR-IOV CNI plugin too in the same cluster: https://github.com/intel/sriov-cni/pull/24 while you cannot reproduce it on yours. There must be something in your cluster that grants additional privileges to the daemonsets that I don't have configured.